packages/gestalt/src/Badge.css

```diff
@@ -0,0 +1,24 @@
+.Badge {
+  composes: antialiased sansSerif letterSpacing from "./Typography.css";
+  composes: fontWeightBold from "./Typography.css";
+  composes: gray from "./Colors.css";
```

Comment:
Nothing wrong with just gray but Im curious why design suggested this since we use other colors in the webapp already. Is the plan to move all those to gray as well? Was this a PDS recommendation?
![image](https://user-images.githubusercontent.com/7877010/51121363-24391a00-17cc-11e9-9b2c-a8cd9f5662a1.png)

Reply:
My original intention was to add more colors (mimicking <Text /> component) but after talking to PDS (Collin and Scott) we decided to use only gray color. One of the biggest reason is that we (PDS/Gestalt) don't have a clear and consistent guidelines for what meaning colors convey. Orange is being used for errors and warnings, green (e.g. Early access) and blue are used for info, success state is rarely (if ever?) conveyed through color. (This is something I would like to discuss in more detail in the future.)
Another reason is that badges already have treatment (rounded border) and don't need an additional color treatment.
As usual, we can start with just gray and add more colors later?

Reply:
Thanks! As @rootvar mentioned, we felt like the original color choices weren't intentional. If anything one could argue that our blue used for education would be fitting but for now secondary gray wasn't controversial so we went with that.